Procfile

```diff
@@ -0,0 +1 @@
+web: node server.js
```

Comment:
Would this allow yarn/nodejs to be able to scan for project file changes during development while also having the game server running or what?

Reply:
This file is actually not needed if we have a valid `scripts.start` entry

We can update the scripts entry to start `webpack --watch` and the server for dev to achieve this goal @DreadKnight